src/app/design/frontend/base/default/layout/omise/gateway.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
```

Comment:
Note, I skipped review this one as I'm not confident much about frontend in Magento.
Will have a look on Magento frontend structure later on.

Reply:
I can confirm this is definitely right... took a lot of Googling and testing to find out how to add CSS correctly